soc/intel/intel_ish/Kconfig.soc

```diff
@@ -0,0 +1,13 @@
+# Intel ISH family selection
+#
+# Copyright (c) 2023-2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_INTEL_ISH
+	bool
+
+config SOC_FAMILY
+	default "intel_ish" if SOC_FAMILY_INTEL_ISH
+
```

Comment:
why having "rsource "*/Kconfig.soc"?
It seems to have */Kconfig.soc included here is better. See other SoC as reference.

Reply:
changed: `Kconfig.soc` moved to `Kconfig`